src/login/test-session-properties.c

```diff
@@ -94,6 +100,37 @@ TEST(set_display) {
         assert_se(isempty(display));
 }
 
+/* Tests org.freedesktop.logind.Session SetTTY */
+TEST(set_tty) {
+        _cleanup_(sd_bus_flush_close_unrefp) sd_bus* bus = NULL;
+        _cleanup_(sd_bus_error_free) sd_bus_error error = SD_BUS_ERROR_NULL;
+        _cleanup_free_ char *tty = NULL, *q = NULL, *path = NULL;
+
+        int fd, r;
+
+        r = getttyname_malloc(STDIN_FILENO, &q);
+        assert_se(r >= 0);
+
+        path = path_join("/dev", q);
+        assert_se(path != NULL);
```

Comment:
nitpick, as per coding style, we don't spell out `== NULL` or `!= NULL` for pointer checks, i.e. change to:

```c
assert_se(path);
```